proto/storage/compliance_operator_v2.proto

```diff
@@ -96,50 +106,50 @@ message ComplianceOperatorScanSettingV2 {
-    string              severity     = 6 [(gogoproto.moretags) = 'search:"Compliance Rule Severity,hidden,store"'];
-    string              description = 7;
-    string              instructions = 8;
-    map<string, string> labels = 9;
-    map<string, string> annotations = 10;
-    google.protobuf.Timestamp     created_time = 11;
-    string standard = 12 [(gogoproto.moretags) = 'search:"Compliance Standard,hidden,store"'];
-    string control = 13;
-    string scan_id = 14 [(gogoproto.moretags) = 'search:"-" sql:"fk(ComplianceOperatorScanV2:id),no-fk-constraint"'];
+    string                    id           = 1 [(gogoproto.moretags) = 'sql:"pk"'];
+    string                    check_id     = 2;
+    string                    check_name   = 3;
+    string                    cluster_id   = 4 [(gogoproto.moretags) = 'search:"Cluster ID,hidden,store" sql:"fk(Cluster:id),no-fk-constraint,type(uuid)"'];
+    CheckStatus               status       = 5  [(gogoproto.moretags) = 'search:"Compliance Check Status,hidden,store"'];
+    RuleSeverity              severity     = 6 [(gogoproto.moretags) = 'search:"Compliance Rule Severity,hidden,store"'];
+    string                    description  = 7;
+    string                    instructions = 8;
+    map<string, string>       labels       = 9;
+    map<string, string>       annotations  = 10;
+    google.protobuf.Timestamp created_time = 11;
+    string                    standard     = 12 [(gogoproto.moretags) = 'search:"Compliance Standard,hidden,store"'];
+    string                    control      = 13;
+    string                    scan_id      = 14 [(gogoproto.moretags) = 'search:"-" sql:"fk(ComplianceOperatorScanV2:id),no-fk-constraint"'];
 }
 
 message ScanStatus {
-    string phase = 1;
-    string result = 2;
+    string          phase    = 1;
+    string          result   = 2;
     repeated string warnings = 3;
 }
 
 // Next Tag: 13
 // Scan object per cluster
 message ComplianceOperatorScanV2 {
-    string              id          = 1 [(gogoproto.moretags) = 'sql:"pk"'];
-    string              scan_name        = 2 [(gogoproto.moretags) = 'search:"Compliance Scan Name,hidden,store" sql:"fk(ComplianceOperatorScanSettingV2:scan_name),no-fk-constraint,index=category:unique;name:scan_unique_indicator"'];
-    string                    cluster_id = 3 [(gogoproto.moretags) = 'search:"Cluster ID,hidden,store" sql:"fk(Cluster:id),no-fk-constraint,type(uuid),index=category:unique;name:scan_unique_indicator"'];
-    repeated string           errors = 4;
-    repeated ProfileShim      profile                 = 5;
-    map<string, string> labels      = 6;
-    map<string, string> annotations = 7;
-    ScanType            scan_type   = 8;
-    NodeRole            node_selector = 9;
-    ScanStatus          status = 10;
-    google.protobuf.Timestamp     created_time             = 11;
-    google.protobuf.Timestamp     last_executed_time        = 12;
```

Comment:
Made `RuleSeverity` an Enum and then formatting while I was in the file.  I'm waffling on whether or not to keep `RuleSeverity` in the storage proto.  If we are going to eventually use it, it will save migration hassle.  If we never use it we will have a dead column.  My inclination is to probably keep it in storage.  Thoughts?